src/pages/Bridge/IssueForm/index.tsx

```diff
@@ -222,17 +218,19 @@ const IssueForm = (): JSX.Element | null => {
     status === STATUSES.PENDING ||
     requestLimitsIdle ||
     requestLimitsLoading ||
-    governanceTokenBalanceIdle ||
     governanceTokenBalanceLoading
   ) {
     return <PrimaryColorEllipsisLoader />;
   }
-  if (requestLimits === undefined || governanceTokenBalance === undefined) {
+
+  if (requestLimits === undefined) {
     throw new Error('Something went wrong!');
   }
 
   if (status === STATUSES.RESOLVED) {
     const validateForm = (value: string): string | undefined => {
+      if (!governanceTokenBalance) return;
```

Comment:
I would rather use explicit `governanceTokenBalance === undefined` to make sure this condition won't be true when account is selected, but balance is 0 (because in that case we display warning there)